File: LeetCode75-sols/Array_String/Python/Reverse_Words_in_a_String.py
class Solution:
    def reverseWords(self, s: str) -> str:
        rev_words = s.split()
        rev_words = rev_words[::-1]
        rev_s = ''
        for word in rev_words:
            rev_s += word
            rev_s += ' '
        rev_s = rev_s.rstrip()
        return rev_s

File: LeetCode75-sols/Array_String/CPP/Reverse_Words_in_a_String.cpp
class Solution {
public:
    string reverseWords(string &s){
        stringstream ss(s);
        string curr_word, answer;
        while(ss >> curr_word){
            answer = curr_word + " " + answer;
        }
        return answer.substr(0, size(answer)-1);  
    }
};
